### PR TITLE
[DEV APPROVED] - 9613 Add cookie message

### PR DESCRIPTION
**Pull request by your**
[TP 9613](https://moneyadviceservice.tpondemand.com/entity/9613-implement-cookie-messaging)

### Technical

This PR adds:

* A controller with an action to manage the creation of a permanent cookie
* A concern with a helper method to check the presence of the above cookie
* A conditional div displayed at the top of the header of every page

Front End 
Added a new javascript component (Cookie) to add the cookie using JS
Cookie is set to expire 20 years in the future as per the Rails version
Checked in Chrome, FF and Safari

### CODE REVIEW

File: app/controllers/cookie_policy_acceptances_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CookiePolicyAcceptancesController < ApplicationController
+  include CookiePolicyAcceptance
```
Comment (MarcoBgn) [inline app/controllers/cookie_policy_acceptances_controller.rb]: Any reason why the module is included both here and in `ApplicationController`?

Comment (your) [reply app/controllers/cookie_policy_acceptances_controller.rb]: @MarcoBgn this was a leftover of a refactoring, thanks for spotting it, removed!

**Comment by scruti** [comment]
+1. BE code LGTM!

**Comment by jiveturkeyJay** [comment]
@rhianwatkins1 Can you have a look at this latest change, just a small change to make the cookie message align to the bottom of the screen in mobile

**Pull request merged**